src/components/Records/Search/Header/HitCount.vue

```diff
@@ -1,6 +1,10 @@
 <template>
-  <div class="mt-2" style="text-align: center;">
-    <p>Displaying {{x}}  to {{y}} of {{ hits }}.</p>
+  <div
+    v-if="hits"
+    class="mt-2"
+    style="text-align: center;"
```

Comment:
Try avoid using style at all costs, especially when there's a vuetify class that does that.
Intead of `style="text-align:center"` use `class="mt-2 text-center"`
(see https://vuetifyjs.com/en/styles/text-and-typography/)